### DISCUSSION TITLE
Vercel - DNS Issue - Beehiiv Subdomain

### DISCUSSION THREAD

**Discussion by sreimer15** [vercel/community#3920] [Help] [answered]
### Page to Investigate

https://vercel.com/guides/pointing-subdomains-to-external-services

### Steps to Reproduce

The docs say it should be relatively simple

`Locate the value and the name arguments requested by the external provider and use them in the DNS UI, replacing name with the name provided, and cname.vercel-dns.com with the value:
`

https://news.srepodcast.com/

Anyone know why this setup doesn't work for vercel? I've done it a few times with google domains and no problems.

Trying to setup my beehiiv newsletter and have vercel hosting, it should be relatively simple but it seems like the wildcard might be affecting things?
<img width="845" alt="Screenshot 2023-09-04 at 1 52 11 AM" src="https://github.com/vercel/community/assets/12318580/e802f7b6-6910-49fe-b8cf-94c52393a053">
<img width="857" alt="Screenshot 2023-09-04 at 1 52 02 AM" src="https://github.com/vercel/community/assets/12318580/65edb7e2-40d2-4249-8bd5-4c877277a18d">
<img width="861" alt="Screenshot 2023-09-04 at 2 13 20 AM" src="https://github.com/vercel/community/assets/12318580/d86e2474-f62d-4743-aa04-db6b1b3e4b92">

**Comment by swarnava** [comment, marked as answer]
Hi,

The cname has to be set to cname.vercel-dns.com not cname.beehiiv.com. Once you update and the DNS propagated fully, it should start working.

Are you pointing to a site at Vercel or for any different service?

**Reply by sreimer15** [reply]
Yeah the site is being hosted by beehiiv which is why I think it wants the DNS record, do I need to setup a redirect of some sort?
I'm not sure what I can ask the beehiiv team to do 

I got this error which makes sense since the deployment is not managed by vercel, but when I switch to the beehiiv cname I get the ssl issue

<img width="709" alt="Screenshot 2023-09-05 at 4 47 29 AM" src="https://github.com/vercel/community/assets/12318580/e1a69cc2-a8af-4526-a9e5-833f89b22917">

**Reply by swarnava** [reply]
Hi,

You need to setup CNAME record not redirect to setup domain. Please contact your domain registrar for more information on this.

**Reply by ClaytonSmith** [reply]
@swarnava I'm having a similar issue however there's nothing my domain registrar can do when Vercel is managing the DNS records. Can you elaborate on `You need to setup CNAME record not redirect to setup domain` and how we could do that on Vercel?
